os/arch/arm/src/armv8-m/up_memfault.c

```diff
@@ -114,36 +121,36 @@ int up_memfault(int irq, FAR void *context, FAR void *arg)
 	/* Dump some memory management fault info */
 
 	(void)irqsave();
-	lldbg("PANIC!!! Memory Management Fault:\n");
-	mfdbg("  IRQ: %d context: %p\n", irq, regs);
```

Comment:
I think we'd better keep mfdbg, because printing too many debug message makes degradation when binary recovery.

Reply:
If we need to keep mfdbg, then better to make a single dbg for all faults  (like fdbg) and use lldbg for lines 129 and 130., and fdbg for all other debug statements.

Reply:
Added fltdbg.